scopes/preview/preview/preview.preview.runtime.tsx

```diff
@@ -80,6 +81,9 @@ export class PreviewPreview {
    * render the preview.
    */
   render = async (rootExt?: string) => {
+    // fit content always.
+    window.document.body.style.width = 'fit-content';
```

Comment:
are we sure that window always exists here?
also, the fit content can break many compositions. we should test this as well